scala_proto/private/scala_proto_default_repositories.bzl

```diff
@@ -0,0 +1,392 @@
+load(
+    "//scala:scala_cross_version.bzl",
+    _default_scala_version = "default_scala_version",
+    _extract_major_version = "extract_major_version",
+    _scala_mvn_artifact = "scala_mvn_artifact",
+)
+load(
+    "@io_bazel_rules_scala//scala:scala_maven_import_external.bzl",
+    _scala_maven_import_external = "scala_maven_import_external",
+)
+
+def scala_proto_default_repositories(
+        scala_version = _default_scala_version(),
+        maven_servers = ["http://central.maven.org/maven2"]):
+    major_version = _extract_major_version(scala_version)
+
+    native.maven_server(
+        name = "scala_proto_deps_maven_server",
+        url = "http://central.maven.org/maven2/",
+    )
+
+    native.maven_jar(
```

Comment:
Thanks for pointing it out @ittaiz! I am planning on deprecating it, but it is currently blocked on finding a satisfying alternative to maven_server. 

Though in this case, I think the following would work instead of both the native.maven_server and native.maven_jar rules:

```
jvm_maven_import_external(
  name = "scala_proto_rules_protoc_jar",
  artifact = "com.github.os72:protoc-jar:3.6.0.1",
  server_urls = ["http://central.maven.org/maven2/"], 
  artifact_sha256 = "44d513ece5a7d9b09f2d865653d82c1d1556900b2450f0e7d629c3bc13ee0e13",
  licenses = ["notice"] # Apache 2.0
)
```